src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -149,27 +149,11 @@ public static String evalXpathExpression(XPathExpression expr,
         return FunctionUtils.toString(expr.eval(evaluationContext));
     }
 
-    /**
-     * @return Data instance built from xml stream or the error message raised during parsing
-     */
-    public Pair<ExternalDataInstance, String> buildExternalDataInstance(InputStream instanceStream) {
```

Comment:
removed as one can directly use `ExternalDataInstance.buildFromRemote` while the error handling is moved to top level classes.